lib/takeaway.rb

```diff
@@ -0,0 +1,62 @@
+require_relative "menu"
+require_relative 'twilio'
+require 'dotenv'
+Dotenv.load
+
+
+class Takeaway
+
+  attr_reader :selected_dishes, :total
+
+  def initialize (menu = Menu.new, twilio = Twilio_sms)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)